### PR TITLE
Support for 3.13

### PR DESCRIPTION
**Pull request by Enforcer**

### CODE REVIEW

**Comment by meadsteve** [comment]
@Enforcer thanks so much for this. I really appreciate you taking the time bumping the build tools and versions. Definitely one of my least favourite tasks maintaining a project.

The build test-failure looks like my CI setup has got a bit stale. I'm fixing that here: #261 .

The failure on 3.13 looks like some test dependency is importing a module that doesn't exist in 3.13 so we might need to bump something in this branch.

File: pyproject.toml [hunk 1]
```diff
@@ -13,17 +13,16 @@ authors = [{ name = "Meadsteve", email = "steve@meadsteve.dev" }]
...
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
   "Topic :: Software Development :: Libraries",
   "Typing :: Typed",
 ]
 dependencies = []
-requires-python = ">=3.7"
+requires-python = ">=3.9"
```
Comment (meadsteve) [inline pyproject.toml:25]: Can we keep this on 3.7. Although it's not formally supported it *should* work so I'd like to have it installable for any user willing to do test coverage themselves.

**Comment by meadsteve** [comment]
If you rebase on the latest master then cherry pick the latest 3 commits here: https://github.com/meadsteve/lagom/pull/263/commits your branch should go green

**Comment by meadsteve** [comment]
@Enforcer this looks pretty good now. Anything else that's making you keep it in draft or shall we get it merged?

**Comment by Enforcer** [comment]
@meadsteve not anymore, forgot to remove `Draft: ` from the title. It's good to go, whenever you're ready :)

**Comment by meadsteve** [comment]
Done! and thank you again for your time.

**Pull request merged**